integrations/opsgenie-integration.mdx

```diff
@@ -3,7 +3,7 @@ title: "Opsgenie"
 description: Opsgenie Integration - Setup & Installation Docs | Middleware
 ---
 
-Integrate with Opsgenie to report incidents direct from Middleware alert section.
+The Opsgenie Integration allows you to create and assign incidents to team members within your Opsgenie organization. Track the total amount of resources consumed by your incident management tool in the Middleware platform.
```

Comment:
remove second "Opsgenie", generally if the pronoun is already used in the sentence it does not need to be used again. Quick ask for you to scan all these and remove redundancy where possible